keras/src/quantizers/quantizers_test.py

```diff
@@ -100,3 +100,759 @@ def test_quantize_and_dequantize(self):
         )
         # A loose assertion due to an expected quantization error
         self.assertAllClose(qdq_values, values, atol=5e-1)
+
+    def _TestOp(
```

Comment:
We can use `@parameterized.named_parameters` and `named_product` to organize similar tests like this one:
https://github.com/keras-team/keras/blob/master/keras/src/ops/nn_test.py#L2355-L2365